### ISSUE TITLE
Tree not visible in IE 9 while using saveState

### ISSUE THREAD

**Issue by rakyat** [mbraak/jqTree#357]
Hi mbraak , Thank you for the great plugin. Im using **saveState** feature with jquery cookie and it works fine in all versions of IE and mozilla in Windows 8 OS. But the tree is not visible in IE and mozilla when the the corresponding webpage with the tree is accessed for the **first time** in other OS.The tree is only visible after opening the developer mode and debug the script.I think there is some problem with cookie.Is there any known issues regarding this? Can u help me with this problem?
By the way I'm adding the tree to a webpage by including another webpage which has some operations with the tree.
Thanks in advance.

**Comment by mbraak** [comment]
How are you including a webpage in another webpage? Are you using an IFrame or something?

**Comment by rakyat** [comment]
I'm developing with java.We can include webpage into another by using an include tag in java.The first webpage includes the json data and the tree is initialized with the data and some nodes are removed from the jtree according to requirement,then this webpage is added to the second webpage after the div for the tree.So that the tree will be visible.

**Comment by mbraak** [comment]
I can help you with javascript, but I'm afraid I cannot help you with the Java part.

**Comment by rakyat** [comment]
No sir there is no issue with the java.The json data,the tree initialization every thing is there when i check the source but the tree is not visible when I add the **saveState** attribute.The tree is visible only after I debug the script.This is only happen when i load the webpage for the **first time** in a new computer(except IE 10,chrome).Once the tree is visible,there is no problem i am guessing that there is some problem creating cookies.If am not using **saveState** it works fine. :(

**Comment by rakyat** [comment]
Sir should i post the code here or i can contact u some other way.I know its confusing. I almost done all the functionality with jqtree but now this new issue has occured and i really need some help and I'm sure it is with browser.Hope u can help..

**Comment by mbraak** [comment]
I tried [this example](http://mbraak.github.io/jqTree/examples/04_save_state.html) in IE9 on Windows 7. It seems to run fine.

What options are you using to initialize the tree? Are you doing something like this?

```js
$('#tree').tree({
  data: data,
  saveState: true
});
```

**Comment by rakyat** [comment]
Sir I'm no expert in jquery . I will explain what i did.

json data
var data = [
    {
        label: 'Tools',
        id: 'Tools',
        children: [
            {label: 'Change Logo', id: 'Change Logo'}
        ]
    },
    {
        label: 'Settings',
        id: 'Settings',
        children: [
            {label: 'Functionality', id: 'Functionality'},
            {label: 'Privileges', id: 'Privileges'},
            {label: 'Document Types', id: 'Document Types'},
            {label: 'Department', id: 'Department'},
            {label: 'Workflow', id: 'Workflow'}
        ]
    },
    {
        label: 'User and Roles',
        id: 'User and Roles',
        children: [
            {label: 'Manage Roles', id: 'Manage Roles'},
            {label: 'Manage Users', id: 'Manage Users'},
            {label: 'Manage Group', id: 'Manage Groups'}
        ]
    }
];

```javascript
<script type="text/javascript">
    var privilegeList = [];
    <s:iterator value="privAccessList" status="stat">    //Java iterator
    privilegeList.push('<s:property></s:property>');    //nodes which are not to be shown in tree are  
    </s:iterator>                                                          // inserted to javascript array
 var $tree = $('#sidebar-data');
    $tree.tree({
        data: data,
        saveState: true
    });

   for (i in privilegeList)
    {
        var node = $tree.tree('getNodeById', privilegeList[i]);
        var next_node = node.getNextSibling();
        var parent_node = node.parent;
        $tree.tree('removeNode', node);
        if (parent_node.children.length === 0)
        {
            $tree.tree('removeNode', parent_node);
        }
    }
</script>
```
All of this are included after 
<div id="sidebar-data">

**Comment by mbraak** [comment]
Is it possible that IE9 switches to compatibility mode when you open the webpage?

**Comment by rakyat** [comment]
Sir...finally it works...you were right ..when i put all these code in the same website it works fine..may be some problem with including the webpage.Sorry to trouble you..I'm grateful for your effort for this plugin and your support for the issues..Thanks... :) :) :)

**Comment by mbraak** [comment]
ok. great
